### PR TITLE
minor: added ssl support to postgres connector

### PR DESCRIPTION
**Pull request by geclos**
# WHAT
Implements ssl support for the postgres connector

### CODE REVIEW

File: packages/connectors/postgresql/src/index.ts [hunk 1]
```diff
@@ -84,36 +94,68 @@ export class PostgresConnector extends BaseConnector {
...
+      return ssl
+    }
+
+    const { sslmode, ca, key, cert, rejectUnauthorized } = ssl
+    const payload = {
+      sslmode,
+      rejectUnauthorized,
+      ca: ca ? readFileSync(ca).toString() : undefined,
+      key: key ? readFileSync(key).toString() : undefined,
+      cert: cert ? readFileSync(cert).toString() : undefined,
```
Comment (andresgutgon) [inline packages/connectors/postgresql/src/index.ts:153]: Where are these files defined?

Comment (geclos) [reply packages/connectors/postgresql/src/index.ts:153]: the client passes the paths to the files themselves

Comment (csansoon) [reply packages/connectors/postgresql/src/index.ts:153]: What if those files do not exist? Is there an understandable error message explaining why it failed?

Comment (geclos) [reply packages/connectors/postgresql/src/index.ts:153]: good point we should handle this gracefully

**Review by csansoon — APPROVED** [review]

**Pull request merged**